src/test/kotlin/no/nav/familie/ef/sak/beregning/BeregningControllerTest.kt

```diff
@@ -58,59 +52,6 @@ class BeregningControllerTest : OppslagSpringRunnerTest() {
         headers.setBearerAuth(lokalTestToken)
     }
 
-    @Test
-    internal fun `Skal klare å inserte ett vedtak med resultatet avslå`() {
```

Comment:
Er denne testen, men med en annen type behandling relevant?

Reply:
Ser mest ut til at den er laget for å teste at lagre blankett-vedtak tillater nesten alt, men jeg har lagt tilake den ene.